components/opc-ua/src/Microsoft.Azure.IIoT.OpcUa.Edge.Publisher/src/Storage/PublishedNodesJobConverter.cs

```diff
@@ -326,6 +327,7 @@ private class OpcNodeModelComparer : IEqualityComparer<(string, OpcNodeModel)> {
                             QueueSize = node.QueueSize ?? standaloneCliModel.DefaultQueueSize,
                             // ToDo: Implement mechanism for SkipFirst.
                             SkipFirst = node.SkipFirst ?? standaloneCliModel.DefaultSkipFirst,
+                            DataChangeFilter = node.DataChangeFilter ?? DataChangeTriggerType.StatusValue
```

Comment:
>node.DataChangeFilter [](http://example.com/codeflow?start=47&length=21)

I think we should still allow `null` value here. This will represent the default behavior as specified here: [https://reference.opcfoundation.org/v104/Core/docs/Part4/7.17.2/](https://reference.opcfoundation.org/v104/Core/docs/Part4/7.17.2/). We should allow the user to decide if he/she goes for the default or StatusValue